haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="
+	ProjectConceptor is a modular application for Haiku and BeOS designed to \
+	view, edit and process information which can be represented by a graph \
+	structure.
+	"
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="
+	2017 Paradoxianer
+	2000-2002 Lee Thomason
+	1991-2000 Be Incorporated
+	"
+LICENSE="
+	BSD (3-clause)
+	MIT
+	MIT (no promotion)
```

Comment:
Re-reading the [source file][0], I don't think this would qualify as "MIT (no promotion)". This is more or less a custom license, so sorry for my earlier suggestion

[0]: https://github.com/Paradoxianer/ProjectConceptor/blob/dfef4b25fbd3dc620e144fef66a0879c4442a8f1/src/app/Tools/ColorTools.h#L4

Reply:
No problem. I can certainly appreciate that you took the pain in trying to get the legal side right.

Reply:
We still have to ship that license. Please add it as the Open Tracker License